src/components/StafferPage/OrganizationDropdown.jsx

```diff
@@ -0,0 +1,65 @@
+import PropTypes from 'prop-types';
+import CreatableSelect from 'react-select/creatable';
+import { Alert } from '@edx/paragon';
+
+const OrganizationDropdown = props => {
+  const {
+    label, input, options, meta: { touched, error },
+  } = props;
+
```

Comment:
Did we not create a similar component for UserPane's course editors? I suggest refactoring the common component and using that common component in both org dropdown and user pane.

Reply:
@zawan-ila

Reply:
We did not create a component for UserPane. We directly used a component provided by react-select [(code ref)](https://github.com/openedx/frontend-app-publisher/blob/master/src/components/SidePanes/UsersPane.jsx#L166)

We did create a component for course tags, however, that component is somewhat different from our current component, in particular, that it fetches options async and renders a different component from react-select than our current one. On top of that, the rendered components(from react-select) take in some significantly different props. As such, while we can still indeed combine both these inside a single component, i am unsure of the benefits of such an approach. 

Please have a look at both of these components ( [Course Tags](https://github.com/openedx/frontend-app-publisher/blob/master/src/components/ReduxFormCreatableSelect/index.jsx#L7) vs [Staffer Organization](https://github.com/openedx/frontend-app-publisher/blob/caeda90867b6ad65e68d90b832e6fe548490ce5f/src/components/StafferPage/OrganizationDropdown.jsx#L5) ) and let me know what you think.

Reply:
> As such, while we can still indeed combine both these inside a single component, i am unsure of the benefits of such an approach.

The benefit is re-usability. Both components are doing similar things. I would not term that as a blocker but having a dedicated component, ideally in openedx paragon, would be beneficial for the community.

Reply:
have refactored reduxformcreatableselect to accommodate our usecases